crates/extensions/c8y_log_manager/src/config.rs

```diff
@@ -71,6 +72,21 @@ impl LogManagerConfig {
     }
 }
 
+fn is_config_dir_exist(config_dir: &Path) -> Result<(), TEdgeConfigError> {
+    match config_dir.parent() {
+        Some(cdir) => {
+            if !Path::new(cdir).is_dir() {
+                return Err(TEdgeConfigError::ConfigDirNotFound(cdir.to_path_buf()));
+            }
+            Ok(())
+        }
+
+        None => Err(TEdgeConfigError::ConfigDirNotFound(
+            config_dir.to_path_buf(),
+        )),
+    }
+}
```

Comment:
Could be moved to `tedge_utils` as a utility function and reused.

Reply:
In fact, there is already a function that exists for verifying the dir path.